schema/fmi3Type.xsd

```diff
@@ -133,8 +133,20 @@ ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 					</xs:complexContent>
 				</xs:complexType>
 			</xs:element>
-			<xs:element name="Boolean" type="fmi3TypeDefinitionBase"/>
-			<xs:element name="String" type="fmi3TypeDefinitionBase"/>
+			<xs:element name="Boolean">
+				<xs:complexType>
+					<xs:complexContent>
+						<xs:extension base="fmi3TypeDefinitionBase" />
+					</xs:complexContent>
+				</xs:complexType>
+			</xs:element>
+			<xs:element name="String">
+				<xs:complexType>
+					<xs:complexContent>
+						<xs:extension base="fmi3TypeDefinitionBase" />
```

Comment:
```suggestion
						<xs:extension base="fmi3TypeDefinitionBase"/>
```

Reply:
Ah yes, thanks. Copy/paste layout problem from the issue.

Reply:
@nickbattle, please check "Allow commits by maintainers" when you open your next PR.

Reply:
@t-sommer Yes, I would if I could, but for some reason when I create PRs via the INTO-CPS fork that option does not appear where it should. We're not sure why. Any idea?